app/views/admin/instances/index.html.haml

```diff
@@ -27,22 +30,20 @@
       %li= filter_link_to t('admin.instances.delivery.failing'), availability: 'failing'
       %li= filter_link_to t('admin.instances.delivery.unavailable'), availability: 'unavailable'
 
-- unless limited_federation_mode?
-  = form_with url: admin_instances_url, method: :get, class: :simple_form do |form|
-    .fields-group
-      - InstanceFilter::KEYS.each do |key|
-        = form.hidden_field key, value: params[key] if params[key].present?
-
-      - %i(by_domain).each do |key|
-        .input.string.optional
-          = form.text_field key,
-                            value: params[key],
-                            class: 'string optional',
-                            placeholder: I18n.t("admin.instances.#{key}")
```

Comment:
Is that comment saying that there are many keys which are currently not used (ie, before this PR) but which because of dynamic interpolation here dont get seen by unused tasks?

If so, maybe pluck out those removals in advance to make this one smaller?

Reply:
Correct, and they only get picked up after this gets merged, hence having to fix here